### PR TITLE
added manager information, opensea api, metrics page, opensea asset components

### PR DESCRIPTION
**Pull request by MAkzent**
image

## TL;DR
- Created page `metrics` that shows all current games and the list of all manager's Assets
  - Not sure if you wanted a list of all assets in the `metrics` page. We can also just show the total asset count per manager and the total asset value. And then add another page called `assets` to list all assets
  - Currently, I'm showing **all** assets of a specific account. We can filter the API by contract address if we want to filter for specific games. (I assumed that manager's addresses will only hold Blackpool NFTs anyways)
- Added `data` folder to store all static related to Blackpool. 
  - Future update: Dynamically fetch Manager data from https://github.com/BlackPool-Finance/managers/blob/main/verticals.json
- Added OpenSea API and created new folder `services` for all APIs
  - Note: API Key is optional
- Because the two current managers have empty wallets, you can use `example manager` in `managers.tsx` to get a list of NFTs.

### CODE REVIEW

**Comment by bneiluj** [comment]
Why not fetching it directly from file? "Future update: Dynamically fetch Manager data from https://github.com/BlackPool-Finance/managers/blob/main/verticals.json"

File: assets/data/managers.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    managers: [
+      {
+        address: '0x90778E7335EF8350a7F2EAe37FC6D549E4ABf095',
+        nickname: 'AJ',
+        verification: true,
+      },
+      {
+        address: '0x84265C1652D6382b6244d958B193ef5895781D5A',
+        nickname: 'Maxmersch',
+        verification: true,
```
Comment (bneiluj) [inline assets/data/managers.tsx:22]: can't be static

File: assets/data/managers.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      },
+      // {
+      //   address: '0x82d92b7bc6dff8775679e6a6c870db3231e6d925',
+      //   nickname: 'Example Manager',
+      //   verification: false,
+      // },
```
Comment (bneiluj) [inline assets/data/managers.tsx:28]: remove logs

**Comment by MAkzent** [comment]
> Why not fetching it directly from file? "Future update: Dynamically fetch Manager data from https://github.com/BlackPool-Finance/managers/blob/main/verticals.json"

We can fetch from `https://raw.githubusercontent.com/BlackPool-Finance/managers/main/verticals.json`.  But I'll need to implement a hook that fetches the data instead of loading it locally.  

What's the reason we have the manager data in a different repo?

**Comment by bneiluj** [comment]
let's just add managers file into this repo then

**Comment by bneiluj** [comment]
Please add your ethereum address as well for rewards + BlackPool's shares.

**Comment by MAkzent** [comment]
> Please add your ethereum address as well for rewards + BlackPool's shares.

0x8928162f714fc0Ae40e3dED7fF5E21eD6E68ad62

Thank you!
